jenkins_jobs/upload_build_artifacts/script.sh

```diff
@@ -12,16 +12,18 @@ else
 fi
 echo  "{'REPO': '$REPOSITORY_FILE_URL', 'BUILD_TIMESTAMP': '${BUILD_TIMESTAMP}', 'BUILD_LOG': '$BUILD_URL', 'BUILD_ID': $BUILD_JOB_NUMBER, 'BUILD_STATUS': '$BUILD_STATUS'}" > STATUS
 
-alias rsync_upload="rsync -e \
-    'ssh -i ${HOME}/.ssh/${UPLOAD_SERVER_USER_NAME}_id_rsa' \
-    --verbose --compress --stats --times --chmod=a+rwx,g+wx,o-"
+rsync_upload() {
```

Comment:
The point of using a function is that you should not need to read its body (implementation) to understand what it does, only its header. I prefer the function than the alias, but I see no problem in adding one or two lines of doc to make code reading easier. We document all functions in Python, why not do the same in shell script? If you think it does not add value, never mind.